config/testgrids/config.yaml

```
@@ -6,10 +6,12 @@ dashboard_groups:
     - gardener-extension-registry-cache
     - gardener-extension-shoot-oidc-service
     - gardener-dependency-watchdog
+    - gardener/gardener-extension-networking-cilium
 
 dashboards:
 - name: gardener-gardener
 - name: gardener-ci-infra
 - name: gardener-extension-registry-cache
 - name: gardener-extension-shoot-oidc-service
-- name: gardener-dependency-watchdog
\ No newline at end of file
+- name: gardener-dependency-watchdog
+- name: gardener/gardener-extension-networking-cilium
```

```suggestion
- name: gardener-extension-networking-cilium
```
This makes the [test fail](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_ci-infra/659/pull-ci-infra-check-testgrid-config/1640719352242114560#1:build-log.txt%3A1).